packages/ui/src/theme/css/component/collection.scss

```diff
@@ -1,12 +1,52 @@
 .amplify-collection {
   flex-direction: column;
-}
-
-.amplify-collection-search {
-  flex-direction: row;
-  justify-content: center;
-}
-
-.amplify-collection-pagination {
-  justify-content: center;
+  &-pagination {
+    justify-content: center;
+    --amplify-components-pagination-current-color: var(
```

Comment:
these tokens are set differently because we want to keep our 010 specificity and these components are composed of other components but we have no way of passing a class directly onto the pieces that we want to theme.  So we are setting the css variable of the pieces we want to theme within the local scope of the component.

Reply:
Wow what a super cool solution!